marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/migration/AllStepsReferencedTest.java

```diff
@@ -0,0 +1,429 @@
+package com.marklogic.hub.hubcentral.migration;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+//import com.marklogic.client.document.JSONDocumentManager;
+import com.marklogic.client.eval.EvalResult;
+import com.marklogic.client.eval.EvalResultIterator;
+import com.marklogic.hub.*;
+import com.marklogic.hub.dataservices.FlowService;
+import com.marklogic.hub.flow.Flow;
+import com.marklogic.hub.flow.FlowInputs;
+import com.marklogic.hub.flow.FlowRunner;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.impl.FlowManagerImpl;
+import com.marklogic.hub.impl.MappingManagerImpl;
+import com.marklogic.hub.mapping.Mapping;
+import com.marklogic.hub.step.RunStepResponse;
+import com.marklogic.hub.step.StepDefinition;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AllStepsReferencedTest extends AbstractHubCoreTest {
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    @Autowired
+    FlowRunner flowRunner;
+
+    @BeforeEach
+    void setUp() {
+        String folderInClasspath = "test-projects/all-steps-referenced";
+        installProjectInFolder(folderInClasspath);
+
+        // copy input directory to ye-olde-project
```

Comment:
installProjectInFolder will do this automatically for you, but the folder has to be named "data" instead of "input"

Reply:
Renamed to "data"

Reply:
I believe you can delete this whole try/catch block then, correct?

Reply:
Yes, I had intended to do that. Just pushed an update with it removed.